src/index.html

```diff
@@ -2,16 +2,182 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()"
+    >
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <label class="label-field" for="surname">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          id="surname"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
          >
```
Добавляй атрибутам отступы